Sparkle/SUFileManager.m

```diff
@@ -39,6 +39,29 @@ static BOOL AuthorizationExecuteWithPrivilegesAndWait(AuthorizationRef authoriza
     return returnValue;
 }
 
+static BOOL SUMakeRefFromURL(NSURL *url, FSRef *ref, NSError **error) {
+
+    char sourcePath[PATH_MAX] = {0};
```

Comment:
probably should be named path instead of sourcePath now that it's in a generalized function?